### PR TITLE
Translatable fixes

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: Admin/Context/Request/AbstractFormValidRequestHandler.php [hunk 1]
```diff
@@ -85,8 +85,13 @@ protected function getRedirectResponse(RedirectableElement $element, Request $re
             return new RedirectResponse($redirectUri);
         }
 
+        $routeParameters = $element->getSuccessRouteParameters();
+        if (true === $request->attributes->has('translatableLocale')) {
+            $routeParameters['translatableLocale'] = $request->attributes->get('translatableLocale');
```
Comment (chives) [inline Admin/Context/Request/AbstractFormValidRequestHandler.php:90]: Why not get the current translatable locale from `LocaleProvider` like in `ElementActionExtension` ?

File: Menu/KnpMenu/ElementVoter.php [hunk 2]
```diff
@@ -72,6 +87,27 @@ public function matchItem(ItemInterface $item): ?bool
         return false;
     }
 
+    private function translatableLocaleMatches(ItemInterface $item): bool
```
Comment (chives) [inline Menu/KnpMenu/ElementVoter.php:90]: Does it really matter if the locale in the menu item match the current one? All urls in main menu are generated using the current locale so why even bother checking? On the other hand the translatable locale menu is totally diffent and diserves custom voter if it's neccessary.

File: Admin/Context/Request/AbstractFormValidRequestHandler.php (hunk 1 above)
Comment (szymach) [reply Admin/Context/Request/AbstractFormValidRequestHandler.php:90]: It is the same thing and this way there is no need to inject additional service to all handlers

File: Menu/KnpMenu/ElementVoter.php (hunk 2 above)
Comment (szymach) [reply Menu/KnpMenu/ElementVoter.php:90]: This is purely for the translation locale switcher. Without this, all languages in the menu are highlighted as active.

**Comment by szymach** [comment]
@rn0

**Review by rn0 — APPROVED** [review]

**Pull request merged**